frescobaldi_app/userguide/outline_configure.md

```diff
@@ -5,7 +5,10 @@ the document text.
 
 You can specify what to search for by entering a list of regular expressions.
 
-In those expressions, `^` matches at the beginning of every line, and `$` 
+Those expressions are grouped in two sections one that matches the whole text (including comments)
```

Comment:
I would add some punctuation, e.g. a colon: "Those expressions are grouped in two sections: one that matches the whole text...".

Reply:
@PeterBjuhr Can you fix this nitpick and merge?

Reply:
Sorry for the delay! Done!